apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -32,7 +36,37 @@
 // identical classes for potential use on other elements if needed.
 
 // ----------------------------------------
-// v 1.0.1
+// v 2.0.0
+// ----------------------------------------
+
+// Sizes
+// Most of the typography elements has a set of different sizes (defined by "-one", "-two", "-three", "-four" prefixes).
+// (Not applicable to Headings and elements without those prefixes)
+// Common rule for all of those sizes are following:
+// 1. "-one" is bigger size.
+// 2. "-two" is a DEFAULT size, used in most of cases.
+// 3. "-three" is smaller size.
+// 4. "-four" is the smallest size.
+
+// Values
+// font-size uses rem
+// - works when the root font-size is 100% which uses the
+// browser default font-size of 16px (3rem = 16 x 3 = 48px)
+
+// margin uses em
+// - sets the margin relative to the parent element rather than the root
+// example:
+// - 0.5rem = 8px margin >>> gets value from root 16px
+// - 0.5em = font-size / 2 >>> gets value from h1 3rem (48px) / 2 = 24px margin
```

Comment:
I don't quite follow the syntax here.

Reply:
Agree, updated. Does it look better to you this way?